### ISSUE TITLE
Create a distinct type for PushCondition::RoomMemberCount::is

### ISSUE THREAD

**Issue by jplatte** [ruma/ruma-events#104]
Currently it is just a string, but it is defined very clearly what it should look like

```rust
        /// A decimal integer optionally prefixed by one of `==`, `<`, `>`, `>=` or `<=`.
        ///
        /// A prefix of `<` matches rooms where the member count is strictly less than the given
        /// number and so forth. If no prefix is present, this parameter defaults to `==`.
        is: String,
```

We could represent this instead as a struct or enum and even easily provide a helper method for whether this condition applies to a given count of room members.

**Comment by jplatte** [comment]
I can write up instructions for how to implement this if anybody is interested.

**Comment by Kinrany** [comment]
> I can write up instructions for how to implement this if anybody is interested.

I am!

Looks like I just need to replace [the type](https://github.com/ruma/ruma-events/blob/7395f940a7cf70c1598223570fb2b731a6a41707/src/push_rules.rs#L137) and fix compilation errors?

Will try on the weekend.

**Comment by jplatte** [comment]
Yes, pretty much! Note that until the weekend, this crate is most likely going to be moved into our new monorepo at https://github.com/ruma/ruma. I will transfer this and other open issues along with it. Also, the new type should encode the actual integer as `js_int::UInt` like almost every other integer in our crates.

**Comment by jplatte** [comment]
I'll assign this issue to you so it's easier to see in the issue list that somebody is working on this.
